packages/components/src/Page/Page.css

```diff
@@ -65,6 +61,10 @@
   width: 100%;
 }
 
+.titleBar.small .subtitle {
```

Comment:
Works on small
![image](https://user-images.githubusercontent.com/39704901/105778007-b1211d00-5f28-11eb-8408-cfe089217360.png)

but we also have a "smaller" that isn't classnamed because it's the "default":
![image](https://user-images.githubusercontent.com/39704901/105778041-c138fc80-5f28-11eb-9992-0fc268ac3443.png)

I _think_ this should get it working for all sizes:

```
.titleBar .subtitle {
  margin-bottom: 0;
}

.titleBar.medium .subtitle,
.titleBar.large .subtitle {
  margin-bottom: large;
}
```

Reply:
(There's a larger discussion about how we've named the size breakdowns, but that's on me as a larger fix because I was the last one messing with these sizes 😅 )